LayoutTests/platform/ios/TestExpectations

```diff
@@ -7477,6 +7477,8 @@ imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-rep
 [ Release arm64 ] fast/events/ios/activating-reset-button-should-not-scroll-page.html [ Skip ]
 [ Release arm64 ] fast/events/ios/activating-submit-button-should-not-scroll-page.html [ Skip ]
 
+webkit.org/b/107356 fast/css/sticky/sticky-top-zoomed.html [ ImageOnlyFailure ]
```

Comment:
It was passing because the scale wasn't actually applied for the test nor the reference at the time of the pixel test?

Reply:
Yep! Already failing on every other platform, since the actual tested content appears to have been broken for a long time.